depends/README.md

```diff
@@ -49,6 +49,10 @@ The paths are automatically configured and no other options are needed unless ta
 
     sudo apt-get install curl librsvg2-bin libtiff-tools bsdmainutils cmake imagemagick libz-dev python3-setuptools libtinfo5 xorriso
 
+Note: You must obtain the macOS SDK before proceeding with a cross-compile.
+The SDK should be placed under `depends/SDKs/`.
```

Comment:
Does this directory need to be created manually?

Reply:
Yep, do you think as-is may imply that the directory `SDKs` should be created, or should I explicitly state that.

Reply:
Thanks; I'd say it, so casual users or people who don't look often in `/depends` won't wonder if they are missing something.

Reply:
adressed in 5e7d1997fce15b3e2fb77e67491237812e1f98d9